doc/libraries.php

```diff
@@ -253,6 +253,11 @@ function libstandard($lib) {
         print ($p ? implode(', ', $p) : '&nbsp;');
     }
 
+    function libstandard_min_level($lib) {
+        print !empty($lib['cxxstd']) ?
+                'C++'.$lib['cxxstd'] : '&nbsp;';
```

Comment:
Can you just print the value of $lib['cxxstd'] as-is? Don't you need to html_encode() it?

Reply:
Just done your feedback @glenfe

Reply:
Thanks @novio. One more thing, if the is no `cxxstd` specified, don't add a "C++ standard minimum level" entry. It won't look good with 99% of libraries having an empty field since their libraries.json won't have it.

Reply:
Ok

Reply:
I have done your feedback @glenfe . So, it will only include `C++ standard minimum level` if `cxxstd` key is found.